test/node-quirks.test.ts

```diff
@@ -17,7 +17,6 @@ const hasNativeFetch = (host.node as VersionInfo).version >= 18;
 
 describe('#fetchHAR (Node-only quirks)', function () {
   let fetchHAR;
-  let initOptions: RequestInit = {};
```

Comment:
looks like you're only removing test coverage in this file, any way you can add a test for that duplex change?

Reply:
Yeah the test is "does this suite still work on Node 18". Without these duplex changes all of these file upload tests fail on Node 18 and 19.